### ISSUE TITLE
Users management i1

### ISSUE THREAD

**Issue by saygunnyc** [Automattic/wp-calypso#70128]
**Project:**
UX fixes on managing "users" while still utilizing Calypso components.

**Issues**

- Does the page/menu name “Users” is apparent enough for customers? Would users look for subscribers underneath? – Can we change this name? –Core WordPress.
- The current user management does not support CSV importing. – We are currently working on enabling this functionality on onboarding.
- The current invite page is too complex for just inviting subscribers – Customers will likely invite more readers/subscribers/followers than team members.
- The current interface prioritizes team members while keeping followers and email followers in the 2nd and 3rd order over the tabs.
- The separation of “Followers” and “Email Followers” may not be intuitive. Users may not understand the difference. – Followers are followers who are also Dotcom users. Do we really need that distinction? Can we indicate that difference more subtly?
- The terminology differences –subscribers v.s. followers might appear to be confusing for some customers.
- Current experience does not support and is not scalable to manage paid subscribers. – Current tipping work on the Reader also might benefit from a list of tips UI.
- Once the user invites the first set of subscribers and then visits Users, the Email Followers/Subscribers tab doesn’t display pending invites, as those go under another tab called Invites. –They might think they have not successfully invited the first batch. Would it make sense to kill the Invites tab and embed pending invitations into the context (each tab)?
- The empty states that show up once the user doesn’t have any team members or followers are not very welcoming or directive in terms of inviting. – Could we directly send them to invite forms?
- There are broken links on tooltips that appear to be linking to support documentation.
- While team members and invites have a detail page associated with the user, followers and email subscribers don’t.
- The CFT for adding Email Subscribers seems out of place within the user management experience.

**p2**
Tailored Use Cases: “User” Management i1" – pbAok1-3rs-p2

**Comment by ebinnion** [comment]
I believe that we've shipped this. Is there outstanding work?

**Comment by saygunnyc** [comment]
Yes, we did! 🚀  I believe we're in a good place UX-wise. A couple of remaining things, improvements for another iteration:

- Moving Subscribers from Users and making it stand alone on the left nav - Using the newer design language. This still needs to have a design.
- The terminology differences –subscribers v.s. Followers might appear to be confusing for some customers. Pablo has this on this post. p9Jlb4-6nS-p2
